src/patternfly/patternfly-date-picker.scss

```diff
@@ -0,0 +1,177 @@
+// stylelint-disable
+.pf-c-date-picker__calendar {
+  &.flatpickr-calendar {
+    // calendar
+    --pf-c-date-picker__flatpickr__calendar--Background: var(--pf-global--BackgroundColor--light-100);
```

Comment:
do you need background on both of them ?

Reply:
Not sure what you mean by both - which other one are you referring to?

Reply:
Do you mean both in the `pf-c-date-picker` component's `__calendar` element, as well as here in these flatpickr overrides?